<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayouttop">
    <item>
     <widget class="QPushButton" name="pushButtonhist">
      <property name="text">
       <string>直方图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonconvolution">
      <property name="text">
       <string>卷积</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonpca">
      <property name="text">
       <string>pca</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>79</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutbottom">
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="widgetLeft" native="true"/>
    </item>
    <item alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QWidget" name="widgetRight" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
